.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -6,23 +6,28 @@ jobs:
 - job: linux
   pool:
     vmImage: ubuntu-16.04
-  timeoutInMinutes: 360
   strategy:
-    maxParallel: 8
     matrix:
-      linux_python3.6.____cpython:
-        CONFIG: linux_python3.6.____cpython
-        UPLOAD_PACKAGES: True
+      linux_64_python3.6.____cpython:
+        CONFIG: linux_64_python3.6.____cpython
+        UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: condaforge/linux-anvil-comp7
-      linux_python3.7.____cpython:
-        CONFIG: linux_python3.7.____cpython
-        UPLOAD_PACKAGES: True
+      linux_64_python3.7.____cpython:
+        CONFIG: linux_64_python3.7.____cpython
+        UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: condaforge/linux-anvil-comp7
-      linux_python3.8.____cpython:
-        CONFIG: linux_python3.8.____cpython
-        UPLOAD_PACKAGES: True
+      linux_64_python3.8.____cpython:
+        CONFIG: linux_64_python3.8.____cpython
+        UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: condaforge/linux-anvil-comp7
+  timeoutInMinutes: 360
+
   steps:
+  - script: |
+         rm -rf /opt/ghc
```

Comment:
This seems a little odd, do you happen to know what's going on here?

Reply:
All of those files are maintained by the rerendering process. I have no idea what the reasons behind the individual bits and pieces are.

Reply:
This is the story behind that line:
https://github.com/conda-forge/conda-smithy/commit/de02148e179bca2b8f249eb96d523da1c1958642

Reply:
Thanks for the pointers!